views/_footer.html

```diff
@@ -1,7 +1,12 @@
 {% block footer %}
-<footer class="site-footer">
-  <div class="footer-wrapper">
-    <p>Footer Text Goes Here</p>
+<footer class="footer">
+  <div class="footer--content">
+    <p>
```

Comment:
@scottnath for what reason? It's not getting any additional styling at the moment. Is it for accessibility?